examples/playbooks/tasks/x.yml

```diff
@@ -1,4 +1,4 @@
-- # nothing, checks bug #849
+-  # nothing, checks bug #849
```

Comment:
This is surprising. It added an extra space?

Reply:
I observed too. We will see. I do have a theory why this happens and I hope we will be able to replicate the behavior with our formatter.